docker/release/build_locally.sh

```diff
@@ -9,6 +9,6 @@ KEYLIME_DIR=${2:-"../../"}
 
 ./generate-files.sh ${VERSION}
 for part in base registrar verifier tenant; do
-  docker buildx build -t keylime_${part} -f ${part}/Dockerfile $KEYLIME_DIR
+  docker buildx build -t keylime_${part} -f ${part}/Dockerfile $KEYLIME_DIR --progress plain ${@:3}
```

Comment:
No we generate custom Dockerfiles using `generate-files.sh` this sets the version.

Reply:
Right, but we do not tag the built image with the correct version, it will always be tagged as `latest`.

I tried it locally here, using a version other than `latest`, and and then it built the base image locally, but tagged it as `latest`. Then we try to build the next image, that depends on `base:VERSION`, but it does not exist locally, so it tries to get it from `docker.io`, then I got a separate issue:

```
#3 ERROR: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

Reply:
Done.